hack/sync_manifests.py

```diff
@@ -119,6 +119,32 @@ def clean_name(name, kind):
         for container in doc["spec"]["template"]["spec"]["containers"]:
             if container["name"] == "manager":
                 container["image"] = "${IMAGE}"
+        # --- Add HA Configuration ---
```

Comment:
Would it make more sense just to put this in the operator?

Reply:
I think that's what my previous commit was doing.

Reply:
The operator code. When we create the deployment.

Reply:
ah, I see your point now. Pragmatically.